README.md

```diff
@@ -6,6 +6,7 @@ The ODF CLI tool provides configuration and troubleshooting commands for OpenShi
 
 - `odf set recovery-profile <profile>`: Set the recovery profile to favor new IO, recovery, or balanced mode with options `high_client_ops`, `high_recovery_ops`, or `balanced`. The default is `balanced`.
 - `odf get recovery-profile`: Get the recovery profile value.
+- `odf rook purge-osd <Id>`: Permanently remove an OSD from the cluster.
```

Comment:
I thought this was the command in the design doc? 
```suggestion
- `odf purge osd <Id>`: Permanently remove an OSD from the cluster.
```

Reply:
I liked how the `purge-osd` goes with `rook`  as the parent sub-command and with time it will be easier to add more under rook I guess, similar to upstream. Thoughts?

Reply:
According to the design doc, we weren't going to focus on commands under component names, to keep more consistency with the pattern: `odf <verb> <sub-verb/subject> <params>`.

Reply:
Updated the design doc with more thoughts on this topic... @subhamkrai I was discussing with @jarrpa and he pointed out that purging an OSD is really specific to ceph and really warrants the component name. So your original suggestion for `odf rook purge-osd` was good, it just took me a while to come around. I would just suggest instead the `ceph` component name since OSDs really belong to ceph, even if we do implement it in the rook operator, to make it:
```
odf ceph purge-osd <id>
```

Reply:
yeah having a component name with `purge-osd` make sense